src/compass/utils/helpers.py

```diff
@@ -253,25 +256,29 @@ def get_burst_bbox(burst_id, burst_db_file=None, burst_db_conn=None):
         burst_db_conn = sqlite3.connect(burst_db_file)
     burst_db_conn.row_factory = sqlite3.Row  # return rows as dicts
 
-    burst_ids = [burst_id] if isinstance(burst_id, str) else burst_id
+    burst_ids = [burst_id] if isinstance(burst_id, str) else \
+        ','.join(burst_id)
 
-    results = []
-    query = "SELECT epsg, xmin, ymin, xmax, ymax FROM burst_id_map WHERE burst_id_jpl = ?"
-    for bid in burst_ids:
-        cur = burst_db_conn.execute(query, (bid,))
-        results.append(cur.fetchone())
+    query = "SELECT epsg, xmin, ymin, xmax, ymax burst_id_jpl FROM burst_id_map WHERE burst_id_jpl = ?"
+    cur = burst_db_conn.execute(query, (burst_ids,))
+    results = cur.fetchall()
 
     if not results:
         raise ValueError(f"Failed to find {burst_ids} in {burst_db_file}")
 
+    n_results = len(results)
+    epsgs = [[]] * n_results
+    bboxes = [[]] * n_results
+    burst_ids = [[]] * n_results
+    for i_result, result in enumerate(results):
+        epsgs[i_result] = result["epsg"]
+        bboxes[i_result]= (result["xmin"], result["ymin"],
+                           result["xmax"], result["ymax"])
+        burst_ids[i_result] = result["burst_id_jpl"]
```

Comment:
Is there a reason to include `burst_id` in the query and regnerate the list of burst IDs, as opposed to using what the user passed in?

Is it just to make sure they're in the same order?

Reply:
You're right - this is redundant.